packages/attention/src/component.tsx

```diff
@@ -91,8 +114,8 @@ export function Attention(props: AttentionProps) {
   return (
     <div
       aria-describedby={ariaDescribedby}
-      role={ariaRole}
-      aria-label={ariaLabel}
+      role={props.tooltip ? 'tooltip' : 'img'}
```

Comment:
Just thinking. Would it still be useful to be able to override the `role` and `aria-label` on the whole component? For example, if users want to set `role="tooltip"` on just one child of the Attention, they'd want to reset the default `role` and `aria-label` on the wrapper. We at least displayed an example like this in the Fabric Elements docs: https://elements.fabric-ds.io/attention.html Wdyt?

Reply:
Hmm good point! I could leave the added props for role and aria-label and do a check that if they are empty we will default to what we have set already in the component and otherwise it will take what the user puts in? :)

Reply:
Sounds good! Just an empty string should be accepted in case they want to reset our default `role` and `aria-label` 😊